scripts/runlogwatch.sh

```diff
@@ -5,9 +5,17 @@ LOG_DIR="/shared/log/ironic/deploy"
 
 mkdir -p "${LOG_DIR}"
 
+# shellcheck disable=SC2034
 python3 -m pyinotify -e IN_CLOSE_WRITE -v "${LOG_DIR}" |
```

Comment:
```suggestion
python3 -m pyinotify --raw-format -e IN_CLOSE_WRITE -v "${LOG_DIR}" |
```
This should disable the coloring of the output.
https://github.com/seb-m/pyinotify/blob/0f3f8950d12e4a6534320153eed1a90a778da4ae/python3/pyinotify.py#L2294

Reply:
I swear I looked for that option...........
thanks!

Reply:
Forgot to commit it?